src/components/collection/Collection.js

```diff
@@ -61,9 +62,9 @@ const Collection = ({ id }) => {
   const parsedDescription = collection?.get('description') || '';
   const tags = collection?.get('settings')?.tags || [];
 
-  // todo: views and likes don't exist
+  // todo: views don't exist
   const views = collection?.get('views');
-  const likes = collection?.get('likes');
+  const likes = parseInt(likeCount?.count, 10);
```

Comment:
I don't think it's normal to have to parseInt it in the frontend. Couldn't it have been dealt with in the backend directly? Most probably you don't even want to have an object {count} but only the number?